Grid.js

```diff
@@ -0,0 +1,168 @@
+import Cell from "./Cell.js";
+
+class Grid {
+    constructor(
+        gridSize,
+        numMines,
+    ) 
+    {
+        this.gridSize = gridSize;
+        this.numMines = numMines;
+        this.cells = [];
+    }
+
+    createGridCells() {
+        this.cells = new Array(this.gridSize);
+    
+        for (let i = 0; i < this.gridSize; i++) {
+            this.cells[i] = new Array(this.gridSize);
+        };
+    
+        for (let row = 0; row < this.gridSize; row++) {
+            for (let col = 0; col < this.gridSize; col++) {
+                let cell = new Cell(
+                    row,
+                    col,
+                    false,
+                    undefined,
+                    false,
+                    false
+                );
+    
+                this.cells[row][col] = cell;
+            };
+            
+        };
+    };
+
+    getCell(row, col) {
+        return this.cells[row][col];
+    }
+
+    changeCellProperty(row, col, prop, value){
+        this.cells[row][col][prop] = value;
+    }
+    
+    assignMines(){  
+        function getRandomInt(max) {
+            return Math.floor(Math.random() * max);
+        }
```

Comment:
this should be extracted (i.e. taken out and put somewhere else) to a helper utility class. something like `GridUtils.js`

Reply:
talked about